packages/web/src/common/utils/genres.ts

```diff
@@ -83,6 +84,7 @@ export const getCanonicalName = (genre: Genre | any) => {
 }
 
 export const GENRES = [
+  Genre.ALL,
```

Comment:
Does this make sense here or should we extend `GENRES` specifically for the filter drawer

Reply:
good question, for now ill just extend it in filter drawer so we dont get issues with web. i was thinking long term we add it and update all our filters, and instead of "null" meaning All, we make this the initialValue for the state too